custom_components/plugwise/models.py

```diff
@@ -23,49 +22,30 @@
     UnitOfVolume,
     UnitOfVolumeFlowRate,
 )
-from plugwise import SmileBinarySensors, SmileSensors, SmileSwitches
+from plugwise.constants import BinarySensorType, SensorType, SwitchType
```

Comment:
These constants should be in the pw-beta `constants.py` file for now.
We can move them later to the backend if needed.

Reply:
Never mind, hadn't seen https://github.com/plugwise/python-plugwise/pull/359